lib/backend/k8s/conversion/conversion_test.go

```diff
@@ -163,14 +163,19 @@ var _ = Describe("Test Pod conversion", func() {
 			// Unknown protocol port is ignored.
 		))
 
+		// Assert the interface name is fixed.  The calculation of this name should be consistent
```

Comment:
Might want to add this same test to the v1.x-series branch too.

Reply:
Ah i thought I had, but I added to the release-1.7 not the 1.x.  Oops.